zebra-consensus/src/parameters/subsidy.rs

```diff
@@ -47,3 +47,114 @@ pub const POST_BLOSSOM_HALVING_INTERVAL: Height =
 ///
 /// Usage: founders_reward = block_subsidy / FOUNDERS_FRACTION_DIVISOR
 pub const FOUNDERS_FRACTION_DIVISOR: u64 = 5;
+
+/// The number of founders reward address intervals, for both mainnet and testnet.
+pub const FOUNDERS_ADDRESS_COUNT: u32 = 48;
+
+/// Mainnet founder adress list as specified in [protocol specification §7.8][7.8]
+///
+/// [7.8]: https://zips.z.cash/protocol/canopy.pdf#foundersreward
+pub const FOUNDERS_REWARD_ADDRESSES_MAINNET: [&str; 48] = [
+    "t3Vz22vK5z2LcKEdg16Yv4FFneEL1zg9ojd",
+    "t3cL9AucCajm3HXDhb5jBnJK2vapVoXsop3",
+    "t3fqvkzrrNaMcamkQMwAyHRjfDdM2xQvDTR",
+    "t3TgZ9ZT2CTSK44AnUPi6qeNaHa2eC7pUyF",
+    "t3SpkcPQPfuRYHsP5vz3Pv86PgKo5m9KVmx",
+    "t3Xt4oQMRPagwbpQqkgAViQgtST4VoSWR6S",
+    "t3ayBkZ4w6kKXynwoHZFUSSgXRKtogTXNgb",
+    "t3adJBQuaa21u7NxbR8YMzp3km3TbSZ4MGB",
+    "t3K4aLYagSSBySdrfAGGeUd5H9z5Qvz88t2",
+    "t3RYnsc5nhEvKiva3ZPhfRSk7eyh1CrA6Rk",
+    "t3Ut4KUq2ZSMTPNE67pBU5LqYCi2q36KpXQ",
+    "t3ZnCNAvgu6CSyHm1vWtrx3aiN98dSAGpnD",
+    "t3fB9cB3eSYim64BS9xfwAHQUKLgQQroBDG",
+    "t3cwZfKNNj2vXMAHBQeewm6pXhKFdhk18kD",
+    "t3YcoujXfspWy7rbNUsGKxFEWZqNstGpeG4",
+    "t3bLvCLigc6rbNrUTS5NwkgyVrZcZumTRa4",
+    "t3VvHWa7r3oy67YtU4LZKGCWa2J6eGHvShi",
+    "t3eF9X6X2dSo7MCvTjfZEzwWrVzquxRLNeY",
+    "t3esCNwwmcyc8i9qQfyTbYhTqmYXZ9AwK3X",
+    "t3M4jN7hYE2e27yLsuQPPjuVek81WV3VbBj",
+    "t3gGWxdC67CYNoBbPjNvrrWLAWxPqZLxrVY",
+    "t3LTWeoxeWPbmdkUD3NWBquk4WkazhFBmvU",
+    "t3P5KKX97gXYFSaSjJPiruQEX84yF5z3Tjq",
+    "t3f3T3nCWsEpzmD35VK62JgQfFig74dV8C9",
+    "t3Rqonuzz7afkF7156ZA4vi4iimRSEn41hj",
+    "t3fJZ5jYsyxDtvNrWBeoMbvJaQCj4JJgbgX",
+    "t3Pnbg7XjP7FGPBUuz75H65aczphHgkpoJW",
+    "t3WeKQDxCijL5X7rwFem1MTL9ZwVJkUFhpF",
+    "t3Y9FNi26J7UtAUC4moaETLbMo8KS1Be6ME",
+    "t3aNRLLsL2y8xcjPheZZwFy3Pcv7CsTwBec",
+    "t3gQDEavk5VzAAHK8TrQu2BWDLxEiF1unBm",
+    "t3Rbykhx1TUFrgXrmBYrAJe2STxRKFL7G9r",
+    "t3aaW4aTdP7a8d1VTE1Bod2yhbeggHgMajR",
+    "t3YEiAa6uEjXwFL2v5ztU1fn3yKgzMQqNyo",
+    "t3g1yUUwt2PbmDvMDevTCPWUcbDatL2iQGP",
+    "t3dPWnep6YqGPuY1CecgbeZrY9iUwH8Yd4z",
+    "t3QRZXHDPh2hwU46iQs2776kRuuWfwFp4dV",
+    "t3enhACRxi1ZD7e8ePomVGKn7wp7N9fFJ3r",
+    "t3PkLgT71TnF112nSwBToXsD77yNbx2gJJY",
+    "t3LQtHUDoe7ZhhvddRv4vnaoNAhCr2f4oFN",
+    "t3fNcdBUbycvbCtsD2n9q3LuxG7jVPvFB8L",
+    "t3dKojUU2EMjs28nHV84TvkVEUDu1M1FaEx",
+    "t3aKH6NiWN1ofGd8c19rZiqgYpkJ3n679ME",
+    "t3MEXDF9Wsi63KwpPuQdD6by32Mw2bNTbEa",
+    "t3WDhPfik343yNmPTqtkZAoQZeqA83K7Y3f",
+    "t3PSn5TbMMAEw7Eu36DYctFezRzpX1hzf3M",
+    "t3R3Y5vnBLrEn8L6wFjPjBLnxSUQsKnmFpv",
+    "t3Pcm737EsVkGTbhsu2NekKtJeG92mvYyoN",
+];
+
+/// Testnet founder adress list as specified in [protocol specification §7.8][7.8]
+///
+/// [7.8]: https://zips.z.cash/protocol/canopy.pdf#foundersreward
```

Comment:
Btw, protocol.pdf and canopy.pdf have been equivalent for a while now. You should prefer referencing protocol.pdf for "the spec corresponding to the latest upgrade that has a stable specification".

Reply:
@oxarbitrage can make that change in this PR, but we also use canopy.pdf elsewhere in the Zebra docs.

I've opened #1226 to follow up those cases.

Reply:
Thanks, opened https://github.com/ZcashFoundation/zebra/pull/1228 for the follow up and changed the ones here at https://github.com/ZcashFoundation/zebra/pull/1170/commits/078af30808d516c761d824412494ca134e6de6ac